src/Compilers/Core/Portable/Syntax/SyntaxToken.cs

```diff
@@ -187,12 +187,12 @@ public object Value
         /// </summary>
         public string ValueText
         {
-            get { return _token != null ? _token.GetValueText() : null; }
+            get { return _token?.GetValueText(); }
         }
 
         public string Text
         {
-            get { return _token != null ? _token.ToString() : string.Empty; }
+            get { return _token?.ToString() ?? string.Empty; }
         }
```

Comment:
I think we should keep the old variant here. We are just deferring to whatever _token.ToString() returns. No need to null-check that.